logouthandler.go

```diff
@@ -0,0 +1,39 @@
+package carp
+
+import (
+	"net/http"
+	"strings"
+	"github.com/sirupsen/logrus"
+)
+
+type LogoutRedirectionHandler struct {
+	logoutUrl string
+	delegate http.Handler
+	logoutMethod string
+	logoutPath string
+}
+
+func NewLogoutRedirectionHandler(configuration Configuration, delegateHandler http.Handler) (http.Handler, error) {
+	return &LogoutRedirectionHandler{
+		logoutUrl: configuration.CasUrl + "/logout",
+		delegate: delegateHandler,
+		logoutMethod: configuration.LogoutMethod,
+		logoutPath: configuration.LogoutPath,
+	}, nil
```

Comment:
If we always return "_, nil" why we have error in the functions signature?

Reply:
Fixed :+1: